src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java

```diff
@@ -70,10 +71,13 @@ public static SscsCaseData convertSyaToCcdCaseData(SyaCaseWrapper syaCaseWrapper
     }
 
     private static String getDwpRegionalCenterGivenDwpIssuingOffice(String benefitTypeCode, String dwpIssuingOffice) {
-        if (dwpIssuingOffice == null) {
+        DwpAddressLookupService dwpAddressLookupService = new DwpAddressLookupService();
+
+        if (CARERS_ALLOWANCE == Benefit.getBenefitByCode(benefitTypeCode)) {
+            dwpAddressLookupService.getDefaultDwpRegionalCenterByBenefitTypeAndOffice(benefitTypeCode);
```

Comment:
What is the point of this? It doesn't return anything...

Reply:
fixed